invert.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include <limits.h>
+
+char *negative_number(int, char *);
+void breakup_bits_into_groups(const char *);
+int invert_last_number(int number, int bits);
```

Comment:
I would make these functions `static` and reorganize the code so that prototypes aren't necessary